src/dsp_tools/utils/excel2json/resources.py

```diff
@@ -135,7 +135,7 @@ def _row2resource(
         try:
             details_df = pd.read_excel(excelfile, sheet_name=name)
         except ValueError as err:
-            raise BaseError(str(err)) from None
+            raise UserError(str(err)) from None
```

Comment:
Here, it depends on how our error system should look like. My implicit assumption is:
 - we use UserError if the user can do something / fix the input data, and if the message is easy enough to be helpful for the user. No stack trace is printed, only the message.
 - we use BaseError if it is handled by the caller
 - we use BaseError if something weird happens which is not handled --> stack trace will be printed

Here we have the 3rd point. If this ever happens, the message will be too complicated for the user - and honestly even for us, because I have never found out what the problem actually is. So I was not able to fix the Excel file.

Now the question is: What should we do in the 3rd case, when something weird happens. My philosophy is: raise a BaseError, so the entire stack trace is printed. The user must get in touch with us anyway, and then we directly have the stack trace (we don't have to search in the logging file first).

What do you think about that?

Reply:
P.S. As I have commented here, I would remove this try-except anyways. Have you seen that comment? What are your thoughts about that?

Reply:
In my mind, the verification did not pass because of some bad user imput, meaning the user caused the error. But if we treat the UserError as you described above then a BaseError makes sense...